docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -32,23 +32,28 @@ As with the previous versions of Ehcache, the canonical way of dealing with `Cac
 include::{sourcedir32}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag=cachemanagerExample]
 ----
 
-<1> Static method `org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder` that returns a new `org.ehcache.config.builders.CacheManagerBuilder`
-    instance;
-<2> Use the builder to register a pre-configured `Cache` to be created when we `.build()` the actual `CacheManager`.
-    The first `String` argument is the alias used to interact with the `Cache` through the `CacheManager`; the second
-    argument is `org.ehcache.config.CacheConfiguration` to configure the `Cache`. We use the static
-    `.newCacheConfigurationBuilder()` method on `org.ehcache.config.builders.CacheConfigurationBuilder` to create a default config;
-<3> Finally, invoking `.build()` returns a fully instantiated, but uninitialized, `CacheManager` we can use;
-<4> Before you start to use the `CacheManager` it needs to be `init()`, which can be done for you by the builder by passing
-    `true` to `build(boolean)`;
-<5> We can retrieve the `preConfigured` aliased `Cache` we declared in step 2. For type-safety, we ask for both key and
-    value types to be passed in. If these differ from the ones we expect, the `CacheManager` throws a `ClassCastException`
-    early in the application's lifecycle. It also guards the `Cache` from being polluted by random types.
+<1> The static method `org.ehcache.config.builders.CacheManagerBuilder.newCacheManagerBuilder` returns a new `org.ehcache.config.builders.CacheManagerBuilder` instance.
+<2> Use the builder to define a `Cache` with alias "preConfigured". This cache will be created when `cacheManager.build()` is invoked on the actual `CacheManager` instance. 
```

Comment:
Isn't it wrong to say that "the cache will be created when `cacheManager.build()` is invoked"? The actual cache doesn't get created on `build()` call. Only the configuration gets built. The actual cache is created in the `cacheManager.init()` call.